backend/controllers/appointments/selectCourtAppointmentRooms.js

```diff
@@ -286,6 +287,29 @@ const selectCourtAppointmentRoomsFactory = ({
         })
       }
 
+      const availableRooms = await availableSlotsService.getAvailableRooms(res.locals, {
```

Comment:
I could be wrong but not sure how well this is going to work. Ideally I think we need to do checks on each of the three time slots: 

- Pre start and end time
- Main start and end time
- Post start and end time

Reply:
It might be a little overkill to use the availableSlotService to do the three checks. Prob best to use the appointmentService and replicate the check we do in addCourtAppointmentService.js

Reply:
looks like using `existingEventsService.getAvailableLocationsForVLB` might be better actually.

Reply:
Ah, yes - that's the one I meant actually. Basically just replicate the initial check in addCourtAppointmentService.js

Reply:
Updated to use `getAvailableLocationsForVLB` from `existingEventsService`.